java/client/src/main/java/glide/api/commands/StreamBaseCommands.java

```diff
@@ -672,4 +673,79 @@ CompletableFuture<Object[][]> xpending(
             StreamRange end,
             long count,
             StreamPendingOptions options);
+
+    /**
+     * Changes the ownership of a pending message.
+     *
+     * @see <a href="https://valkey.io/commands/xclaim/">valkey.io</a> for details.
+     * @param key The key of the stream.
+     * @param group The consumer group name.
+     * @param consumer The group consumer.
+     * @param minIdleTime The minimum idle time for the message to be claimed.
+     * @param ids An array of entry ids.
+     * @return An <code>array</code> of message entries with the format <code>
+     *     [[id, ["entry", "data"]], ...]</code> that are claimed by the consumer.
```

Comment:
A bit more js style for a map
```suggestion
     *     {entryId : [key, value], ...}</code> that are claimed by the consumer.
```